### DISCUSSION TITLE
国内用户常见问题

### DISCUSSION THREAD

**Discussion by Yue-Yang** [Yue-Yang/ChatGPT-Siri#29] [Announcements]
# 碰到无法找到 choices 的情况：

ChatGPT Plus 订阅和使用API是不同服务，它们是分开计费的。

- 首先排查下网络，需要能访问 https://api.openai.com  这个网址，国内需要代理
- 其次，你API key 可能无效了：检查下邮箱，看看是不是ban了。
- 再者，检查 OpenAI API 后台看是否有余额，是否余额过期，免费试用结束需要绑卡：https://platform.openai.com/account/usage 价格表：https://openai.com/pricing

一般就可以解决了。

# 无法用 Siri 启动

先打开设置-》Siri 回答-〉选自动，或者朗读。钩上“始终显示你对Siri说的话”，可以方便查看识别是否精准。

# 找不到文件

需要打开「设置」- Apple ID - iCloud - iCloud 云盘 功能，才能使用保存到 txt 文件功能。

# 安装不了1.2

我的测试设备是iOS16。据有用户反应，在低版本iOS上，1.2 版本安装不了，你可以选择低版本安装。

# 其他

- 如何安全使用API key（英文）：https://help.openai.com/en/articles/5112595-best-practices-for-api-key-safety
- API keys: https://platform.openai.com/account/api-keys
- API 用量: https://platform.openai.com/account/usage
- API 价格: https://openai.com/pricing
- ChatGPT API 常见问题: https://help.openai.com/en/articles/7039783-chatgpt-api-faq

**Comment by Druchii236641237** [comment]
我全局代理了, 我是 APPLE 13 PM, IOS16系统, 能访问https://platform.openai.com/account/api-keys 这个地址, 也没有任何提示 (没有"碰到无法找到 choices 的情况"). 但是转圈圈转了很久以后, 告诉我 "出错了"

**Reply by qq516249940** [reply]
Your agent's country does not support this service
